docs/guards/guards.md

```diff
@@ -55,9 +55,9 @@ Although you probably want to use magic methods in most cases, it might be usefu
 
 ## Custom Guards
 
-As you have seen in the previous section, each guard is a class in the `Uniform\Guards` namespace. To write your own guards you can either provide them through a PHP package and Composer or implement them in the traditional way as a Kirby plugin. The plugin may be a file `site/plugins/uniform-guards.php` that will be automatically loaded by Kirby.
+As you have seen in the previous section, each guard is a class in the `Uniform\Guards` namespace. To write your own guards you can either provide them through a PHP package and Composer or implement them in the traditional way as a Kirby plugin. The plugin may be a file `site/plugins/custom-guards/index.php` that will be automatically loaded by Kirby.
```

Comment:
Maybe `site/plugins/uniform-custom-guards` instead?

Reply:
This sentence is wrong - we can't make it easily load Uniform namespace without Composer. It's gone